memcached.h

```diff
@@ -141,6 +141,19 @@
 #define APPEND_NUM_STAT(num, name, fmt, val) \
     APPEND_NUM_FMT_STAT("%d:%s", num, name, fmt, val)
 
+/** Item client flag conversion */
+#define FLAGS_CONV(iar, it) (__extension__ ({ \
```

Comment:
instead of masking C errors with __extension__ and returning the flag value, could this instead take a pointer to flag and set it directly?